tutorials/getting-started-netcore.rst

```diff
@@ -0,0 +1,186 @@
+Getting Started on .NET Core
+============================
+
+.. note::
+
+    Still on the .NET Framework? Check out our alternate guide to :doc:`/tutorials/getting-started` on the .NET Framework.
+
+You can be up and running with Freya in minutes! This quick guide will take you
+from zero to Hello World, using some of the high level building blocks that
```

Comment:
This formatting looks a little odd - I think elsewhere we just have docs use a natural wrap? Don't think it'll do much with RST, but seems different!